### PR TITLE
Refactor #1

### PR DESCRIPTION
**Pull request by krzysiekprzekwas**

### CODE REVIEW

File: GameServer/MapObjects/MapCircle.cs [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ﻿using GameServer.Models;
 using Newtonsoft.Json;
 using System;
-using System.Collections.Generic;
-using System.Text;
```
Comment (tomejzen) [inline GameServer/MapObjects/MapCircle.cs]: style++

**Comment by tomejzen** [comment]
Let's discuss about the DI when as soon as you will find time for it.
If we don't change time variables to TimeSpan, can we append to each variable name "Miliseconds" so everything is clear.

**Comment by tomejzen** [comment]
user-select: none;  for fullscreen button

File: GameServer/Physics/PhysicsEngine.cs [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
-                player.Speed *= 1 - Config.PLAYER_DECCELERATION;
+                player.Speed *= 1 - _config.PlayerDecceleration;
 
                 var speedVector = player.Speed + GetSpeedFromPlayerInput(player);
 
                 UpdatePlayerPosition(player, speedVector);
             }
 
-            GameState.Instance.Bullets.RemoveAll(b => b.Speed.Length() < Config.MIN_BULLET_SPEED);
+            GameState.Instance.Bullets.RemoveAll(b => b.Speed.Length() < _config.MinBulletSpeed);
```
Comment (tomejzen) [inline GameServer/Physics/PhysicsEngine.cs]: Are we still using Instance?

In this method we are iterating through bullets 2 times. Please create method for bullet like ShouldBeRemoved and then

Bullets.RemoveAll(b => b.ShouldBeRemoved());

**Comment by tomejzen** [comment]
PhysicsEngine.cs (line 27)
                bullet.Speed *= 0.99;

We don't want to use magic numbers here. Also the client will extrapolate bullets on front and we need to share this variable with client.

**Comment by tomejzen** [comment]
PhysicsEngine.cs (line 17)
player.Speed *= 1 - Config.PLAYER_DECCELERATION;

I created this terrible line. Please remove subtracting here.

File: GameServer/GameServer.csproj [hunk 3]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <PublishWithAspNetCoreTargetManifest>false</PublishWithAspNetCoreTargetManifest>
+    <TypeScriptToolsVersion>3.0</TypeScriptToolsVersion>
```
[the remark this answers is not in the document]
Comment (krzysiekprzekwas) [reply GameServer/GameServer.csproj]: I Changed signalR from static file to libman. Due to that change signalr is being rebuild during wwwroot restore and it is written in TS. I can try to obey this by removing src folder from libman reference however it seems like dumbe effort.

**Pull request merged**